db/seeds.rb

```diff
@@ -27,6 +29,22 @@
 
 puts ">>> SEEDING ENVIRONMENT: #{Rails.env}"
 
+if Region.all.empty?
```

Comment:
@patmbolger can you point me to more info about using `.blank?` on an ActiveRecord class?  or explain that?  

Given a scenario where I have no regions in the db, in a rails console:
```
>> Region.blank?
false
>> Region.all.blank?
  Region Load (0.4ms)  SELECT "regions".* FROM "regions"
true
```